R/init.R

```diff
@@ -117,6 +117,15 @@ build_site_meta <- function(pkg = ".") {
     )
   }
 
+  write_to_inst <- pkg$write_meta_to_inst %||% FALSE
```

Comment:
I think it might be better to start a new top-level key. Maybe `meta$deploy$package_yaml`?

Reply:
Okay, so a "deploy" group of options at the top level - can do

Reply:
This should be resolved by my latest chagnes - I've named it `meta$deploy$install_metadata`